packages/components/src/templates/next/components/complex/LogoCloud/LogoCloud.tsx

```diff
@@ -0,0 +1,45 @@
+import type { IsomerSiteProps } from "~/types"
+import { isExternalUrl } from "~/utils"
+import { ImageClient } from "../Image"
+
+interface Image {
+  src: string
+  alt: string
+}
+interface LogoCloudProps extends Pick<IsomerSiteProps, "assetsBaseUrl"> {
+  images: Image[]
+  title?: string
+}
+export const LogoCloud = ({
+  images: baseImages,
+  title,
+  assetsBaseUrl,
+}: LogoCloudProps) => {
+  const images = baseImages.map(({ src, alt }) => {
+    const transformedSrc =
+      isExternalUrl(src) || (assetsBaseUrl === undefined && !!src)
+        ? src
+        : `${assetsBaseUrl}${src}`
+
+    return { src: transformedSrc, alt }
+  })
+  return (
+    <div className="px-10 py-12">
+      <div className="flex flex-col gap-4">
+        <p className="prose-headline-base-medium self-center text-base-content-light">
+          {title ?? "With support from these agencies"}
```

Comment:
btw `title` is required and won't be falsy? just checking if this is just a preventive measures?

Reply:
yes defensive! reason why is because this is written to disk and read back (+ done manually)